crates/sb_workers/event_worker.js

```diff
@@ -0,0 +1,41 @@
+const primordials = globalThis.__bootstrap.primordials;
+const { SymbolAsyncIterator } = primordials;
+const core = globalThis.Deno.core;
+
+class SupabaseEventListener {
+    async nextEvent() {
+        try {
+            return await core.opAsync("op_event_accept");
+        } catch (e) {
+            console.log(e);
+        }
+    }
+
+    #buildEvent(event) {
+        const rawEvent = event['Event'];
+        const eventType = Object.keys(rawEvent)[0];
+        return {
+            deployment_id: "",
```

Comment:
- [ ] Implement `DeploymentId`, `Region`, `ExecutionId`

Reply:
Can we make these fields configurable via event worker?